src/txacme/test/test_client.py

```diff
@@ -344,6 +344,7 @@ def test_unexpected_update(self):
             self.assertThat(
                 client.register(reg2),
                 failed_with(IsInstance(errors.UnexpectedUpdate)))
+        succeeded(client.stop())
```

Comment:
This constructs a matcher then does nothing with it; I think you want an `assertThat` call here.

Reply:
True. Thanks.